### PR TITLE
Add Sioyek.app v1.0.0

### PR DESCRIPTION
**Pull request by alecshaw**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/sioyek.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  url "https://github.com/ahrm/sioyek/releases/download/v#{version}/sioyek-release-mac.zip",
+      verified: "github.com/ahrm/sioyek/"
+  name "Sioyek"
+  desc "PDF viewer designed for reading research papers and technical books"
+  homepage "https://sioyek.info/"
+
+  container nested: "build/sioyek.dmg"
+
+  app "sioyek.app"
```
Comment (neutric) [inline Casks/sioyek.rb:13]: Thank you for your contribution to `homebrew-cask` @alecshaw!

Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

Comment (alecshaw) [reply Casks/sioyek.rb:13]: Thanks! Hope this sorts it.

Comment (neutric) [reply Casks/sioyek.rb:13]: Looking good! Thanks!

**Comment by bevanjkay** [comment]
Thanks @alecshaw! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request merged**